Makefile

```diff
@@ -13,7 +13,7 @@ ORG := jenkins-x
 ORG_REPO := $(ORG)/$(NAME)
 RELEASE_ORG_REPO := $(ORG_REPO)
 ROOT_PACKAGE := github.com/$(ORG_REPO)
-GO_VERSION := 1.18.4
+GO_VERSION := 1.19.2
```

Comment:
Please change this to: https://github.com/jenkins-x/jx/blob/d06edf36217af6eb6ffd7a209245933dabd4cdb3/Makefile#L18